Blake3Core/Blake3.cs

```diff
@@ -30,26 +29,37 @@ public class Blake3 : HashAlgorithm
         Stack<ChainingValue> _chainingValueStack;
         Output _output;
 
-        protected private Blake3(Flag defaultFlag, ReadOnlySpan<uint> key)
+        protected private Blake3(Flag defaultFlag, ReadOnlySpan<uint> key, int? outputSizeInBits)
```

Comment:
Why not just use a default parameter value instead of futzing around with a nullable int?